<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 251, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="baseimg">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Target Photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>580</y>
      <width>231</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>   Start CreatingÔºÅ</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>570</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>568</width>
       <height>528</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>20</y>
      <width>570</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>568</width>
       <height>528</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>580</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 26pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string> Rank:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>81</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 16pt System;
color: rgb(0, 170, 255);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Ultra-low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Middle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ultra-high</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="subimg">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>580</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&quot;Mosaics&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 26pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>Please Choose:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomBig1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/plus.png</normaloff>
      <normalon>:/icon/plus_2.png</normalon>
      <selectedon>:/icon/plus_2.png</selectedon>:/icon/plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomSmall1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/minu.png</normaloff>
      <normalon>:/icon/minu_2.png</normalon>
      <selectedon>:/icon/minu_2.png</selectedon>:/icon/minu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomBig2">
    <property name="geometry">
     <rect>
      <x>1015</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/plus.png</normaloff>
      <selectedon>:/icon/plus_2.png</selectedon>:/icon/plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomSmall2">
    <property name="geometry">
     <rect>
      <x>1095</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/minu.png</normaloff>
      <selectedon>:/icon/minu_2.png</selectedon>:/icon/minu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
